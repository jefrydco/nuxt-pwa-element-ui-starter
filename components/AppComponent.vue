<template>
  <div id="component" style="margin-bottom: 24px">
    <el-date-picker
      v-model="example"
      type="datetimerange"
      range-separator="To"
      start-placeholder="Start date"
      end-placeholder="End date"
    >
    </el-date-picker>
  </div>
</template>

<script>
export default {
  data() {
    return {
      example: null
    }
  }
}
</script>
