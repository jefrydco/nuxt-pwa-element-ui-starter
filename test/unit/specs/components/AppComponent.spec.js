import { shallowMount, createLocalVue } from "@vue/test-utils";
import ElementUI from "element-ui";

import AppComponent from "~/components/AppComponent";

const localVue = createLocalVue();
localVue.use(ElementUI);

describe("App Component", () => {
  it("renders to match snapshot", () => {
    const wrapper = shallowMount(AppComponent, {
      localVue
    });
    expect(wrapper).toMatchSnapshot();
  });
});
