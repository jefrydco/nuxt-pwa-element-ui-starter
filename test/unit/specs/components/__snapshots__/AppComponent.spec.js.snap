// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Component renders to match snapshot 1`] = `
<div id="component" style="margin-bottom: 24px;">
  <el-date-picker-stub appendtobody="true" offset="0" boundariespadding="5" arrowoffset="35" startplaceholder="Start date" endplaceholder="End date" clearicon="el-icon-circle-close" name="" clearable="true" id="" editable="true" align="left" rangeseparator="To" validateevent="true" type="datetimerange"></el-date-picker-stub>
</div>
`;
